doc/releases/migration-guide-4.1.rst

```diff
@@ -89,6 +89,15 @@ Display
 Enhanced Serial Peripheral Interface (eSPI)
 ===========================================
 
+Entropy
+=======
+
+* BT HCI based entropy driver now directly sends the HCI command to parse random
+  data instead of waiting for BT connection to be ready. This is helpful on
+  platforms where the BT controller owns the HW random generator and the application
+  processor needs to get random data before BT is fully enabled.
+  (:github:`79931`)
```

Comment:
I wrote it here because this was a change compared to the previous behavior. I mean, if the final user is already successfully using this driver then he won't notice any difference, but now they can use it also in other scenarios. I thought that `release-notes` was mostly for new features.
Perhaps I misinterpreted the difference between `migration-guide` and `release-notes`. Do you suggest to move it anyway?

Reply:
Yeah there's probably some space for interpretation but I think the migration guide should be for stuff where the user has to take an action, like change something on the application side. @kartben? Anyway I don't think it's super important. One thing you may mention though is that some bluetooth functionalities now depends on the `mbedtls` module rather than `tinycrypt`, this is useful if anyone is running a project with a `name-allowlist` filter project list.